HATE/Shuffle.cs

```diff
@@ -936,15 +853,13 @@ private static void SwapUndertaleBackground(UndertaleBackground kv, UndertaleBac
                 SwapUndertaleBackground(kv, nv);
                 return;
             }
-            var value = list[k];
-            list[k] = list[n];
-            list[n] = value;
+            (list[k], list[n]) = (list[n], list[k]);
```

Comment:
Did you want to turn the if above to a switch statement too?

Reply:
Wanted to. I just went and fixed stuff based on what IDE suggested, but i didn't go through everything yet since there's a lot and figured for ease of review i'll do smaller chunks first.
So yeah, thats planned to do later.